packages/fxa-profile-server/lib/routes/subscriptions.js

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const Joi = require('joi');
+
+module.exports = {
+  auth: {
+    strategy: 'oauth',
+    // TODO: using email scope for now, may change someday
```

Comment:
I hope `someday` is `today` or at least `before this PR lands`.

Reply:
I think this was something @bbangert mentioned - i.e. to re-use the profile email scope for access to read subscription status. We can change it though, if there's a different existing scope or we want to invent a new one.

Reply:
FWIW: If only in the interests of getting this landed to unblock other work, I'm thinking it might be okay to punt this to a follow up issue to be resolved before the launch of the first RP for this new subscriptions stuff.

Reply:
Created a follow-up issue for this - easy enough to change when we want to change it, as long as we do it before the first subscription RP joins on

https://github.com/mozilla/fxa/issues/882